source/read-write-concern/read-write-concern.rst

```diff
@@ -332,17 +332,13 @@ same as :javascript:`writeConcern: {journal: false}`.
 Inconsistent WriteConcern
 -------------------------
 
-Drivers SHOULD raise an error when an inconsistent ``WriteConcern`` is
```

Comment:
Perhaps you were waiting until merge time, but just in case: I think this warrants bumping the spec version to the next minor number, updating the last modified date, and adding a line to the changelog.

Reply:
LGTM-ed, but don't forget to bump this info above.